plugins/plugin-sass/plugin.js

```diff
@@ -136,6 +136,8 @@ module.exports = function sassPlugin(_, {native, compilerOptions = {}} = {}) {
         input: contents,
         env: native ? undefined : npmRunPath.env(),
         extendEnv: native ? true : false,
```

Comment:
thats so odd, `env: native ? undefined : npmRunPath.env(),` above should be putting your npm `run`/`bin` path into the execa environment.  Maybe we don't need both then?

Reply:
In this case the npm run path is not the same as the path that contains the snowpack.config.js file. That's why passing `localDir` is needed.